services/provider/server/server.go

```diff
@@ -734,7 +735,8 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 		}
 	}
 
-	clientProfile := util.CalculateMD5Hash(req.StorageClaimName)
```

Comment:
where else was this being used in this func? I see new code is added using it but no change in existing code, is that the expectation?

Reply:
This was also being used in VolumeReplicationClass, This probably is a bug as well that was missed. 
If we have a different clusterID in VolumeReplicationClass it might cause problems. Yes, the expectation is that the clusterID should be the same for VRC, VGSC, VSC, and SC and should be the clientProfileName

Reply:
ack, so this changes implicitly fixes other bug?

Reply:
Yes right

Reply:
resolve.